.travis.yml

```diff
@@ -23,29 +23,37 @@ compiler:
 - gcc
 - clang
 
+env:
+- PIP='pip'
+
 matrix:
   include:
   - os: linux
     env:
     - QPID_PROTON_CMAKE_ARGS='-DCMAKE_BUILD_TYPE=Coverage'
+    - PIP='pip'
```

Comment:
You don't need this as it is the same as the default above.

Reply:
I didn't have it first and the non matrix'd builds didn't succeed; see [buildlog](https://travis-ci.org/RoddieKieley/qpid-proton/jobs/325304860). Was there something I missed maybe?

Reply:
I think we may be talking about the opposite case:
- I'm suggesting that this line which part of a specific matrix build is not necessary because there is a line in the non matrix case.
- I think you are saying that not having the non matrix line doesn't work. I agree with this.